packages/expo-image/android/src/main/java/expo/modules/image/ExpoImageView.kt

```diff
@@ -49,10 +49,27 @@ class ExpoImageView(context: ReactContext, private val requestManager: RequestMa
         }
     }
   }
+  private var propsChanged = false
   private var loadedSource: GlideUrl? = null
   internal var sourceMap: ReadableMap? = null
-  var blurRadius: Int? = null
-  var fadeDuration: Int? = null
+  internal var blurRadius: Int? = null
+    set(value) {
+      field = if (value != null && value > 0) {
+        value
+      } else {
+        null
+      }
```

Comment:
I'm not sure if `valueIsValid` or `isValueValid` 😓 
```suggestion
      field = if (isValueValid(value)) value else null
```

Reply:
Or
```suggestion
     field = value.takeIf { it != null && it > 0 }
```

Or even shorter:
```suggestion
      field = value?.takeIf { it > 0 }
```

Regarding the naming, I'm more for `isValueValid`, but I think my suggestion is short enough and creating another function doesn't make sense.

Reply:
I'm not sure if `takeIf` is good solution in this situation, because I would like to assign null if value is zero - to avoid handling special case when value is zero, but not null - in our case null is equivalent to zero.

Reply:
It does the exact think you want:

> Returns `this` value if it satisfies the given predicate or `null`, if it doesn't.

Docs: https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/take-if.html